js/reunion.js

```diff
@@ -0,0 +1,51 @@
+import { getMeeting } from './api/meetings.js';
+
+var url = new URL(window.location);
+
+const searchParams1 = new URLSearchParams(url.search);
+const param = searchParams1.get('meetingId');
```

Comment:
Se podría usar `meetingId` en el nombre de la variable, `param` no es específico
```suggestion
const meetingId = searchParams1.get('meetingId');
```